documentation/sphinx/source/bulkdump.rst

```diff
@@ -0,0 +1,96 @@
+##############################
+BulkDump (Dev)
+##############################
+
+| Author: Zhe Wang
+| Reviewer: Michael Stack, Jingyu Zhou
+| Audience: FDB developers, SREs and expert users.
+
+
+Overview
+========
+In a FoundationDB (FDB) key-value cluster, every key-value pair is replicated across multiple storage servers. 
+The BulkDump tool is developed to dump all key-value pairs within the input range to files.
+Note that when the input range is large, the range is splitted into smaller ranges.
+Each subrange data is dumped to a file at a version. All data within a file is at the same version. However, different files' version can be different.
+
+Input and output
+----------------
+When a user wants to start a bulkdump job, the user provides the range to dump and the path root to dump the data. 
+The range can be any subrange within the user key space (i.e. " " ~ "\\xff").
+Dumping the data of the system key space and special key space (i.e. "\\xff" ~ "\\xff\\xff\\xff") is not allowed.
+The path root can be either a blobstore url (TBD) or a path of a file system.
+Given the input range, if the range is large, the range is splitted into smaller ranges.
+Each subrange is dump at a version to a folder. In particular, the folder is organized as following:
+
+1. <rootLocal>/<relativeFolder>/<dumpVersion>-manifest.txt (must have)
```

Comment:
The format has some problem, as the rendered output looks like `//-manifest.txt (must have)`. You might need to escape `<` and `>`.